### ISSUE TITLE
BUG: track which remote got tagged

### ISSUE THREAD

**Issue by CJ-Wright** [ergs/rever#44]
One may want to run a trial run tag against a remote that is not production (I certainly did as I didn't want to mess things up on my first outing with rever). Currently this won't work since if the log is not git tracked then it will know that the tag was successful (at least against the other remote). Also we might aim to be as verbose as possible in the logs anyway just for the sake of provenance.

**Comment by scopatz** [comment]
Yes, this is a good point.  I was copying the repo over to new dir on the file system and messing with that. There is probably a more elegant solution to trying rever out and performing dry runs.
